ocs_ci/utility/utils.py

```diff
@@ -2661,11 +2662,15 @@ def create_rhelpod(namespace, pod_name, timeout=300):
 
     label_pod_security_admission(namespace=namespace)
 
-    # TODO: This method should be updated to add argument to change RHEL version
+    if rhel_version >= 8:
+        RHEL_POD_YAML = constants.RHEL_8_7_POD_YAML
+    else:
+        RHEL_POD_YAML = constants.RHEL_7_7_POD_YAML
+
     rhelpod_obj = helpers.create_pod(
         namespace=namespace,
         pod_name=pod_name,
-        pod_dict_path=constants.RHEL_7_7_POD_YAML,
+        pod_dict_path=RHEL_POD_YAML,
```

Comment:
Related to the previous comment:
```suggestion
        pod_dict_path=rhel_pod_yaml,
```

Reply:
done